src/templates/base.html

```diff
@@ -136,23 +142,91 @@
     </script>
       {% endcomment %}
       {% comment %}
-    Dynammically change where the search bar sends a GET request to based on
-    the selected value of the query type dropdown menu.
+      Dynammically change the autocomplete optins and where the search bar sends a GET
+      request to based on the selected value of the query type dropdown menu.
       {% endcomment %}
-      <script>
+      {# disable djlint for this section because the linter breaks the template tags #}
+      <!-- djlint:off --><script>   // "djlint:off" turns the linter off for this section of code
         document.addEventListener('DOMContentLoaded', function() {
           const search_bar = document.getElementById('search-bar');
           const query_type = document.getElementById('query-type');
 
-          query_type.addEventListener("change", function() {
+          // by default, query autocompletes game names
+          {% get_games as games %}
```

Comment:
I really like your use of comments to explain the purpose of code! If you want to more detailed with comments, you could related to important logical steps. For example, here you could add a comment next to 
{% get_games as games %}
 stating that we are:
// Fetching game names for autocomplete

Though, this is not strictly necessary and your use of comments already is really good.

Reply:
I agree, adding an explicit comment saying line 182 is a query would be quite nice.